queue/remote_ordered.go

```diff
@@ -96,33 +83,42 @@ func (q *remoteSimpleOrdered) Next(ctx context.Context) amboy.Job {
 						"edges": dep.Edges(),
 						"dep":   dep.Type(),
 					}))
+				q.dispatcher.Release(ctx, job)
 				q.addBlocked(job.ID())
 				continue
 			case dependency.Blocked:
-				// this is just an optimization; if there's one dependency its easy
-				// to move that job *up* in the queue by submitting it here. there's a
-				// chance, however, that it's already in progress and we'll end up
-				// running it twice.
 				edges := dep.Edges()
-				grip.Debugf("job %s is blocked. eep! [%v]", id, edges)
+				grip.Debug(message.Fields{
+					"job":       id,
+					"edges":     edges,
+					"dep":       dep.Type(),
+					"job_type":  job.Type().Name,
+					"num_edges": len(edges),
+					"message":   "job is blocked",
+				})
+
 				if len(edges) == 1 {
+					// this is just an optimization; if there's one dependency its easy
+					// to move that job *up* in the queue by submitting it here. there's a
```

Comment:
Remove asterisks? Putting formatting in a comment seems weird. Also capitalize for full sentences.